core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -243,6 +244,26 @@ public boolean equals(Object o) {
    */
   boolean isClosed();
 
+  /**
+   * Convenience method for {@link #checkConsistency(AlluxioURI, CheckConsistencyPOptions)} with
+   * default options.
+   *
+   * @param path the root of the subtree to check
+   * @return a list of inconsistent files and directories
+   */
+  List<AlluxioURI> checkConsistency(AlluxioURI path) throws IOException;
```

Comment:
I don't think `checkConsistency` belongs in the `FileSystem` API. It isn't very useful programmatically, and is really just used by the `checkConsistency` shell command, which could instead make a 1-off client and doesn't need any sort of pooling.

Reply:
Could we move this directly into the `CheckConsistencyCommand` then?

Reply:
sgtm